via/set.go

```diff
@@ -49,3 +49,34 @@ func Reset(address string) error {
 
 	return errors.New(fmt.Sprintf("Incorrect response for command. (Response: %s)", resp))
 }
+
+func SetVolume(address string, volumec string) (string, error) {
+	defer color.Unset()
+	color.Set(color.FgYellow)
+
+	var command ViaCommand
+	command.Command = "Vol"
+	command.Param1 = "Set"
+	command.Param2 = volumec
+
+	log.Printf("Sending volume set command to %s", address)
+
+	resp, err := SendCommand(command, address)
+	if err != nil {
+		return "", errors.New(fmt.Sprintf("Error in setting volume on %s", address))
+	}
+
+	// Error handling - Handle with care
+	// if command returns vol|set|Error1 - value is outside the bounds of 0-100
+	// if command returns vol|set|Error2 - no value set for volume
+	if strings.Contains(resp, "Error1") {
+		log.Printf("Volume command error - volume value %s is outside the bounds of 0-100", volumec)
```

Comment:
Just for conformity's sake I'd set the console color to these errors to red. 

`log.Printf(color.HiRedString(<stuff>))`